apps/designer/app/canvas/shared/breakpoints.ts

```diff
@@ -64,6 +65,15 @@ const useBreakpointDelete = () => {
         }
 
         deleteCssRulesByBreakpoint(rootInstance, breakpoint.id);
+
+        // delete breakpoint styles
+        // reversed order to splice without breaking index
+        for (let index = styles.length - 1; index >= 0; index -= 1) {
+          const style = styles[index];
+          if (style.breakpointId === breakpoint.id) {
+            styles.splice(index, 1);
+          }
+        }
       }
```

Comment:
should be a utility like deleteCssRulesByBreakpoint with a test

Reply:
This is basically filterMutable, can be extracted in utilities if you want

Reply:
yeah